examples/text_classification/iterable_train.py

```diff
@@ -0,0 +1,157 @@
+import os
+import logging
+import argparse
+
+import torch
+import sys
+import io
+
+from torchtext.datasets import text_classification
+from torch.utils.data import DataLoader
+
+from model import TextSentiment
+
+from torchtext.vocab import build_vocab_from_iterator
+from torchtext.data.utils import ngrams_iterator
+from torchtext.data.utils import get_tokenizer
+from torchtext.data.utils import build_iterable_dataset_from_iterator
+from torchtext.utils import unicode_csv_reader
+from torchtext.vocab import build_vocab_from_iterator
+
+from tqdm import tqdm
+
+
+def generate_batch(batch):
+
+    def generate_offsets(data_batch):
+        offsets = [0]
+        for entry in data_batch:
```

Comment:
This could be replaced by `np.cumsum`

Reply:
We don't want a hard numpy dependency, but I agree that'd be prettier